src/index.html

```diff
@@ -7,8 +7,50 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
```

Comment:
<img width="1370" alt="Screenshot 2023-08-15 at 09 32 07" src="https://github.com/mate-academy/layout_moyo-header/assets/53228259/940bcc12-cf70-4207-825d-004087957d6a">

Reply:
well, here I hava not such big attributes and there are only two attributes so my implementation matches thr example

Reply:
fixed